lib/gems/pending/appliance_console/key_configuration.rb

```diff
@@ -45,11 +46,15 @@ def ask_question_loop
     end
 
     def activate
+      if key_exist? && force
+        backup_key
+      end
+
```

Comment:
I was thinking that putting existing `v2_key` in temp file first and moving it to back-up only when fetch successful would be sufficient (and removing temp file in `ensure`